SPEC.md

```diff
@@ -594,7 +596,8 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
             "name": "work",
             "kind": "host",
             "source": "/opt/tenant1/work",
-            "readOnly": true
+            "readOnly": true,
+            "fulfills" "/work"
```

Comment:
fulfills was removed. Looks like the definition below did not get removed in a previous change.